moz_telemetry/sandboxes/heka/analysis/moz_telemetry_parquet_schema.lua

```diff
@@ -0,0 +1,213 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+# Mozilla Telemetry Parquet Schema Documentation
+
+Generates parquet schema documentation for each docType in the data stream.
+
+## Sample Configuration
+
+```lua
+filename = "moz_telemetry_parquet_schema.lua"
+message_matcher = "Uuid < '\003' && Fields[docType] != NIL" -- slightly greater than a 1% sample
+ticker_interval = 60
+preserve_data = false
+```
+
+## Sample Output
+```
+
+-- Type -> Fields[docType] -> Fields[sourceVersion] -> parquet schema.
+The number in brackets is the number of occurrences of each dimension in the sample.
```

Comment:
Is the total size of the sample available? If not I can file a bug for an enhancement.